deeprank_gnn/NeuralNet.py

```diff
@@ -20,15 +18,27 @@
 _log = logging.getLogger(__name__)
 
 
-class NeuralNet(object):
-
-    def __init__(self, database, Net,
-                 node_feature=['type', 'polarity', 'bsa'],
-                 edge_feature=['dist'], target='irmsd', lr=0.01,
-                 batch_size=32, percent=[1.0, 0.0],
-                 database_eval=None, index=None, class_weights=None, task=None,
-                 classes=[0, 1], threshold=None,
-                 pretrained_model=None, shuffle=True, cluster_nodes='mcl', transform_sigmoid=False,
+class NeuralNet():
+
+    def __init__(self, # pylint: disable=too-many-arguments, too-many-locals, too-many-branches
```

Comment:
All the arguments with the comment `noqa` are not used. Should we remove them?

Reply:
Oh sorry, these variables are actually used with `self.__setattr__(k, v)`. I'll leave this as it is for now, but in the future I think it's better to modify the attributes assignation following the best practices.